src/components/Dashboard/TodoList.vue

```diff
@@ -1038,6 +1041,14 @@ export default {
     /** Closes current panel or opens new panel. */
     togglePanel (index: number) {
       this.panel = (this.panel === index) ? null : index
+    },
+
+    /** Determine the Annual Report Due date for a given filing. */
+    arDueDate (filingYear: number, foundingDate: Date) {
+      let dueDate = new Date(foundingDate)
+      dueDate.setFullYear(filingYear)
+      dueDate.setDate(dueDate.getDate() + 61)
```

Comment:
Please add comment re: why 61.

Reply:
Absolutely, there is a reason!

Reply:
Maybe i was overthinking this 😆  
The business rule was: "The due date is 60 days AFTER the anniversary date for a given year", so i was thinking the day-of anniversary wasn't counted but it never was. Updated to 60 days and added comment in next commit!